packages/frontend-web/src/app/scenes/Dashboard/index.ts

```diff
@@ -21,11 +21,9 @@ import { compose } from 'redux';
 import { combineReducers } from 'redux-immutable';
 import { createStructuredSelector } from 'reselect';
 import { CategoryModel } from '../../../models';
-import { getCurrentUser, getIsAdmin } from '../../auth';
-import { getWebsocketState, IAppStateRecord } from '../../stores';
-import {
-  getCategories,
-} from '../../stores/categories';
+import { IAppStateRecord } from '../../stores';
+import {getCategories} from '../../stores/categories';
```

Comment:
nit: keep formatting the same, here and other files (spaces w.r.t. braces); VS Code has a function for auto-formatting, and there are some other tools that help with this too.